pylarky/larky.py

```diff
@@ -0,0 +1,8 @@
+import subprocess
+
+
+def evaluate(script, input_file, output_file):
+    print(subprocess.call(['larky-runner',
```

Comment:
1. Would be nice to have some exception(s) exposed for the cases when something goes wrong during the evaluation process (larky-runner exited with non 0 status code).
2. We probably want to read stderr so the exception mentioned above would have some info that can be used by clients of this lib.